src/status_im2/contexts/chat/composer/style.cljs

```diff
@@ -125,3 +125,10 @@
    {:transform [{:translate-y translate-y}]
     :opacity   opacity}
    {}))
+
+(def mention
+  {:color                 (colors/theme-colors colors/primary-50
+                                               colors/primary-60)
+   :background-color      colors/primary-50-opa-10
```

Comment:
I think this color might be wrong - it should be based on the chat color or the users custom color?
<img width="297" alt="Screenshot 2023-07-28 at 12 13 29" src="https://github.com/status-im/status-mobile/assets/22799766/f166d01c-f2b6-4661-affa-abf51045087d">